contracts/rem.auth/src/rem.auth.cpp

```diff
@@ -67,6 +67,49 @@ namespace eosio {
       cleanupkeys();
    }
 
+   void auth::execaction(const name &account, action& act, const block_timestamp &action_timestamp,
+                         const public_key &pub_key, const signature &signed_by_pub_key, const name &payer)
+   {
+      bool is_payer = bool(payer);
+      name payer_name = is_payer ? payer : account;
+      if (is_payer) { require_auth(payer_name); }
```

Comment:
why do we need **payer** in this action?

Reply:
@roman-tik, done.